frontend/chess-heritage-explorer/style.css

```
@@ -0,0 +1,749 @@
+/* ==========================================================================
+   INCREDIBLE INDIA EXPLORER - CHESS HERITAGE EXPLORER STYLESHEET
+   ========================================================================== */
+
+@import url("../../styles.css");
```

Stylelint (17.14.0)

[error] Expected "url("../../styles.css")" to be ""../../styles.css"" (import-notation)

(import-notation)